### PR TITLE
Spec file cleanup

### PR DESCRIPTION
**Pull request by lemenkov**

### CODE REVIEW

**Comment by lemenkov** [comment]
@runlevel5 @leigh123linux take a look at this one

File: vcmi.spec [hunk 1]
```diff
@@ -9,14 +9,12 @@ URL:            https://vcmi.eu/
 Version:        1.5.2
 Release:        0%{?dist}
```
Comment (runlevel5) [inline vcmi.spec:10]: @lemenkov 
```suggestion
Release:        1%{?dist}
```

**Review by runlevel5 — APPROVED** [review]
LGTM 👍

**Comment by runlevel5** [comment]
@lemenkov I have manually cherry-picked your commits to the rpmfusion's git repository

Thanks for the contribution

P/S: This github repo is just a mirrored repository

**Pull request merged**